### PR TITLE
This PR adds a bug report template

### PR DESCRIPTION
**Pull request by Asmit2952**
Signed-off-by: Asmit2952 <asmitbm2952002@gmail.com>

### CODE REVIEW

**Review by samj1912 — APPROVED** [review]

File: .github/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+### **Additional context**
+Add any other context about the problem here.
+
+<hr/>
+
+<!--- Do not remove or change this in the issue description. Only update the details above this. --->
+**Note:**
+* If you want to work on an issue, you should check if it has already been assigned to anyone. **If the issue is free** you can comment `/assign` to get the issue assigned to you.
+* If you are raising a new issue and want to work on it then also you should comment `/assign` under the issue to get it auto assigned.
+* Please **refrain from** adding labels to your issue/pull-request on your own. It is the job of the Project Admin and the Mentors to review your issue/pull-request and add labels accordingly.
```
Comment (jromero) [inline .github/bug_report.md:26]: I believe `/assign` is a function of PROW which our project doesn't use. Can we change this to state that something like "if you would like to be assigned to it please comment on the issue".

**Review by jromero — CHANGES_REQUESTED** [review]

**Comment by samj1912** [comment]
@Asmit2952 ping ^

**Comment by Asmit2952** [comment]
> @Asmit2952 ping ^

Sorry for replying late and making changes as I didn't get any notification regarding this.

**Comment by Asmit2952** [comment]
Any updates regarding this PR? @jromero @samj1912

**Review by jromero — APPROVED** [review]

**Pull request merged**